spec/Notification-spec.ts

```diff
@@ -1,45 +1,46 @@
+import {expect} from 'chai';
 import * as Rx from '../dist/cjs/Rx';
-declare const expectObservable;
 
+declare const expectObservable;
 const Notification = Rx.Notification;
 
 /** @test {Notification} */
 describe('Notification', () => {
   it('should exist', () => {
-    expect(Notification).toBeDefined();
-    expect(typeof Notification).toBe('function');
+    expect(Notification).exist;
+    expect(Notification).to.be.a('function');
```

Comment:
was aware but didn't tempted quite especially given repo is somewhat inactive state. Updating assertion was actually wasn't that hard, just simple replace.